### ISSUE TITLE
Postgresql - nullpointerexception when displaying arrays 

### ISSUE THREAD

**Issue by Cathychristopher** [dbeaver/dbeaver#6313]
#### System information: 
- macOS Mojave 10.14.5
- DBeaver version: 6.1.3
- Java version =1.8.0_144
- Additional extensions: none

#### Connection specification:
- Database name and version: PostgreSQL 9.5.15 on x86_64-pc-linux-gnu, compiled by gcc (GCC) 4.8.2 20140120 (Red Hat 4.8.2-16), 64-bit
- Driver name: 
    - Java JDBC 4.1 (JRE 7+) driver for PostgreSQL database
    - net.postgis:postgis-jdbc:RELEASE 2.2.1
- Do you use tunnels or proxies (SSH, SOCKS, etc)? SSL mode: required

#### Describe the problem you're observing:
When I select an array type using a constant `select '{1,2,3}'::int[];`, the value displays as "NullPointerException" rather than the array value. It's only an issue when using a constant, not an issue when selecting array type column from a table like so;

```sql
select '{1,2,3}'::int[] into temp tbl_a;
select * from tbl_a;
```

#### Steps to reproduce, if exist:
`select '{1,2,3}'::int[];`

#### Include any warning/errors/backtraces from the logs

Error Backtrace
```java
java.lang.NullPointerException
	at org.jkiss.dbeaver.model.DBUtils.getEntityFromMetaData(DBUtils.java:1804)
	at org.jkiss.dbeaver.model.DBUtils.getEntityFromMetaData(DBUtils.java:1792)
	at org.jkiss.dbeaver.model.impl.jdbc.data.JDBCCollection.makeCollectionFromArray(JDBCCollection.java:197)
	at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCArrayValueHandler.getValueFromObject(JDBCArrayValueHandler.java:62)
	at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreArrayValueHandler.getValueFromObject(PostgreArrayValueHandler.java:82)
	at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreArrayValueHandler.getValueFromObject(PostgreArrayValueHandler.java:1)
	at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCComplexValueHandler.fetchColumnValue(JDBCComplexValueHandler.java:50)
	at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreArrayValueHandler.fetchColumnValue(PostgreArrayValueHandler.java:49)
	at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCAbstractValueHandler.fetchValueObject(JDBCAbstractValueHandler.java:49)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetDataReceiver.fetchRow(ResultSetDataReceiver.java:122)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.fetchQueryData(SQLQueryJob.java:755)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeStatement(SQLQueryJob.java:484)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.lambda$0(SQLQueryJob.java:407)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:146)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:405)
	at org.jkiss.dbeaver.ui.editors.sql.execute.SQLQueryJob.extractData(SQLQueryJob.java:849)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:2776)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:98)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:146)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:96)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:102)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
```

Error Log - Workspace Log
```  
eclipse.buildId=unknown
java.version=1.8.0_144
java.vendor=Oracle Corporation
BootLoader constants: OS=macosx, ARCH=x86_64, WS=cocoa, NL=en_AU

Framework arguments:  -keyring /Users/<user>/.eclipse_keyring
Command-line arguments:  -os macosx -ws cocoa -arch x86_64 -keyring /Users/<user>/.eclipse_keyring

org.jkiss.dbeaver.model
Warning
Mon Jul 22 18:19:25 AEST 2019
Can't read column 'int4' value

```

**Comment by serge-rider** [comment]
Very similar to #6312

**Comment by zachsa** [comment]
Bump.

This works via the psql client: `select cast ('{1}' as int[]);`. But throws a NullPointerException from DBeaver.

**Comment by uslss** [comment]
The issue is fixed in 6.1.4 in the scope of #6173 
Verified
